prow/config.yaml

```diff
@@ -52,8 +52,9 @@ presubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:0.11
+      - image: gcr.io/k8s-testimages/bazelbuild:0.12
         args:
+        - "--repo=k8s.io/$(REPO_NAME)"
```

Comment:
The original intent was to make the job definitions even easier to copy/paste (say, between `kubernetes` and `kubernetes-security`) without having to tweak, but if you think it sacrifices readability, I'm pretty ambivalent.